core/cap-0033.md

```diff
@@ -181,6 +189,7 @@ enum OperationType
 
 struct SponsorFutureReservesOp
 {
+    // Source account will sponsor future reserves for this account
```

Comment:
This comment should lead with the field so that it is clear who the field is in the comment. Because it leads with the source account that is not in this struct it is easy to think the source account is the sponsoredID here.

This might be a bit wordy for you, but:
```suggestion
    // SponsoredID references the account for which future reserves will be sponsored by the source account of the operation this SponsoreFutureReservesOp is specified within.
```

Reply:
Fixed, did something close to what you wrote. Thanks!